### DISCUSSION TITLE
posts/cf-bpb-vpn/

### DISCUSSION THREAD

**Discussion by giscus** [leegical/leegical.github.io#12] [Announcements]
# posts/cf-bpb-vpn/

无需域名，无需 SSL，通过 Cloudflare 和 BPB Panel，搭建一个永久免费且高速的免费 VPN。结合 Cloudflare 实现优选订阅永久免费节点订阅，为使用（singbox-core 和 xray-core）的跨平台客户端提供配置。

https://www.haoyep.com/posts/cf-bpb-vpn/

**Comment by Kuki93** [comment]
## 牛，跟着操作一遍成功了！！！
### 这个是什么原理呀？谁提供的免费服务器？这个稳定吗

**Reply by leegical** [reply]
互联网大善人Cloudflare，稳定了很长时间

**Comment by zhouuuuuuj** [comment]
请教大佬，为什么我打不开KV，点KV不是跳到概述，就是显示“403 | 您无权查看此页面。”

**Reply by aveOuO** [reply]
直接在概述中创建page 之后再去kv创建命名空间试试 我也是点kv跳到概述 后面创建page之后就能点kv了

**Reply by leegical** [reply]
我是在创建了一个代理后重新写的教程，没注意到是否存在创建kv顺序的问题。已在文章中更新相关提示。

**Comment by bugcat9** [comment]
这也太强了，不过这样部署的会不会过几天就被封了

**Reply by leegical** [reply]
自用流量小，不会那么快被封的

**Comment by 3706878228dp** [comment]
请问：我到部署完成后，点击继续处理项目。或者返回到概述中，选择部署的项目，点击访问站点，这一步，点击之后就跳转到了：看来我们遇到了一些内部问题。
我们的团队已收到通知。如果您希望提供帮助，请在下面告诉我们发生的情况。
页面，这是什么问题？

**Reply by leegical** [reply]
关掉代理，浏览器匿名模式试试

**Comment by ions-bit** [comment]
部署成功后访问面板让输入密码不是创建密码，怎么办大佬

**Reply by leegical** [reply]
是不是设置了其他参数？如果不行的话删除重新部署一下试试

**Comment by yyes-eipai** [comment]
作者你好呀，今天我尝试访问项目网址： https://你的项目地址.pages.dev/panel时，就显示you have been blocked。这咋整呢，谢谢！

**Reply by aveOuO** [reply]
我也遇到了 我的解决办法是删掉这个page然后重新部署一次 但是名字不能和上一个page相同 然后就可以了

**Reply by zeroChen00** [reply]
我也是用了一段时间报错 Sorry, you have been blocked。然后删除重新部署，避开bpb关键词，结果又报错Error 1101

**Reply by aveOuO** [reply]
> 我也是用了一段时间报错 Sorry, you have been blocked。然后删除重新部署，避开bpb关键词，结果又报错Error 1101

https://github.com/bia-pain-bache/BPB-Worker-Panel/issues/508#issuecomment-2502335455
我今天用了这个issue里别人提供的worker.js部署就成功解决了1101

**Reply by zeroChen00** [reply]
但是1102了qwq

**Reply by leegical** [reply]
目前确实不行了，等开发者想办法绕过cloudflare的审查吧

**Reply by leegical** [reply]
3.0新版本部署OK了

**Comment by huazinet** [comment]
您好方案是不是和谐了，昨天能用，今天打开bpb后台直接Error 1101

**Reply by aveOuO** [reply]
可以去bpb项目的issue看看有提供 worker.js 部署的时候通过worker部署不用github的page部署就可以了

**Reply by leegical** [reply]
已更新混淆方案，可以避免1101错误

**Comment by zeroChen00** [comment]
PROXYIP填写具体ip后，面板扫描ip部分变成了 download scanner...

**Comment by lonely-4** [comment]
看我的[Starry-Sky-World/BPB-Worker-Panel-Chinese](https://github.com/Starry-Sky-World/BPB-Worker-Panel-Chinese)
中文汉化

**Comment by gib886** [comment]
部署完成显示：无法访问存储库，这可能会导致部署失败。求教大佬们这个怎么解决啊？

**Comment by gib886** [comment]
还显示BPB面板出现问题请设置UUID和木马密码，这个是什么问题？

**Reply by leegical** [reply]
更新bpb3.0版本的教程，重新设置一下变量

**Reply by gib886** [reply]
小白一枚，麻烦问下怎样重置变量？

**Reply by leegical** [reply]
删掉，重新添加

**Reply by gib886** [reply]
根据上面的新教程添加好了，但是v2ray也导入订阅链接了，但是感觉没用呢还是访问不了需要的网站

**Reply by leegical** [reply]
用NORMAL SUB的链接

**Reply by gib886** [reply]
服务器里面怎么什么都没有呢？

**Comment by zeroChen00** [comment]
复活了！好耶！

**Comment by gib886** [comment]
大佬，运行了三天目前显示错误1101了，这个怎么解决？

**Reply by leegical** [reply]
已更新混淆代码方案，可以解决1101错误

**Reply by gib886** [reply]
混合代码设置完了显示找不到网址

**Reply by gib886** [reply]
混合代码部署完成后点击二级域名显示找不到***页，大佬这是什么情况？

**Reply by leegical** [reply]
注意看[文章](https://www.haoyep.com/posts/cf-bpb-vpn/#bpb-%E9%9D%A2%E6%9D%BF)，二级域名后要加上`/panel`哦

**Reply by gib886** [reply]
二级域名一直加了/panel，找不到网页

**Comment by zhlf0420** [comment]
太牛了！我在网上找了各种办法，只能Works搭建，绑定自定义域名不方便。用你的方法一次就成功了

**Comment by shimarin880** [comment]
喋，能不能出个视频教程，小白看不懂

**Comment by zhlf0420** [comment]
3天前BPB更新了内容，然后自动获取的代码经常出错了。
Run wget -O origin.js https://raw.githubusercontent.com/bia-pain-bache/BPB-Worker-Panel/refs/heads/main/unobfuscated/worker.js          
--2025-02-24 02:19:12--  https://raw.githubusercontent.com/bia-pain-bache/BPB-Worker-Panel/refs/heads/main/unobfuscated/worker.js
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.109.133, 185.199.110.133, 185.199.111.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.109.133|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2025-02-24 02:19:12 ERROR 404: Not Found.

**Reply by leegical** [reply]
原仓库改变了加密前的worker.js文件路径，拉不到文件所以报错了。现已同步最新路径。不过也没必要紧跟版本，能用就行啦

**Comment by Tubunu** [comment]
为什么访问有一些网站会时不时的提示Error 1034

**Reply by leegical** [reply]
是不是没配置 **PROXYIP** ？

**Reply by Tubunu** [reply]
配置填的是：cdn.xn--b6gac.eu.org。有时候有这个问题有时候又不会

**Comment by 1135165369** [comment]
因为  PROXYIP：设置找不到合适的ip折腾了好几天   有幸搜到老师的文章  按老师的教程  一遍成功  感恩老师的分享。备注 小白朋友 最重要的是老师分享的这条“PROXYIP：去这里随机选择一个代理 IP，或者你也可以直接把代理 IP 设置为 cdn-b100.xn--b6gac.eu.org 或 bpb.yousef.isegaro.com”。

**Comment by Residual-wolf** [comment]
你好，想问一下为什么我cloudflare一直是部署失败

**Reply by Residual-wolf** [reply]
好了，自己重新混淆后上传文件就正常了

**Reply by millo0420** [reply]
想知道你如何混淆代码的。我这两天部署也是失败

**Reply by xilost** [reply]
v2rayNl里面更新订阅提示
 BPB->Arg_TimeoutException
BPB->无效的订阅内容
为什么呢？

**Reply by Residual-wolf** [reply]
混淆的话我是自己写的混淆代码，跑了一下就好了，我怀疑是博主这个示例用的人太多导致的，v2的问题我不太清楚，我个人用的是NG，更新了就能使用

**Comment by xilost** [comment]
v2rayNl里面更新订阅提示
BPB->Arg_TimeoutException
BPB->无效的订阅内容
为什么呢？

**Reply by leegical** [reply]
在浏览器直接访问你的订阅链接，看能不能成功访问。如果不能访问，那应该是部署BPB项目出了问题。

**Comment by RaidenMaySakura** [comment]
大佬，现在BPB改成通过release发布源码了，自动混淆脚本拉取不到，能否更新一下？

**Reply by leegical** [reply]
BPB在[Release v3.2.3](https://github.com/bia-pain-bache/BPB-Worker-Panel/releases/tag/v3.2.3)中指出，直接未经混淆的代码很容易让Cloudflare检测到并报1101错误。因此本文不再更新混淆脚本。实际上，已经成功部署的BPB脚本也一直是可以正常运行的，没必要跟进最新版本。如果确实想自己混淆，可以拉BPB库，按照[workflows](https://github.com/bia-pain-bache/BPB-Worker-Panel/blob/main/.github/workflows/build.yml)过程编译混淆。
>1- None obfuscated files removed from releases forever, due to code exposures to Cloudflare (possible 1101 errors in future). So please choose a Cloudflare account which haven't returned 1101 before.
2- If you are using Workers deployment and got 1101, please delete the worker and change subdomain from Workers & Pages > Subdomain from right hand toolbar. It will take a few minutes to apply new subdomain.

**Comment by rowcisi** [comment]
大佬，我的配置突然不能用了，proxyIP都能ping通，但是访问不了，bpb面板也是能ping不能访问，加自定义域名也不行，看了下github和CF状态都正常，实在是找不出问题！求救！！！

**Reply by rowcisi** [reply]
但是搭代理后却能访问，关掉代理就又不行了

**Comment by Kimhungtse** [comment]
必需登录上来感谢你，今天刚好想上yt查点资料，发现用不了了，去年11月份的时候也是差不多这种情况，都解决了，厉害，思路清晰，内容简单明了，这次自带的地址给墙了，必需需要一个域名去解析转发才能登录上面板。

**Comment by runtu406076262** [comment]
你好，按照操作后，进入bpb面板还是1101，这咋整

**Comment by wym100171113** [comment]
现在貌似找不到源代码了

**Comment by XINYI55** [comment]
各位大佬们，请问我用BPP搭建的面板，用苹果的小火箭可以正常使用。v2rayNG不能订阅，显示失败，但是用其他翻墙软件打开VPN后，v2rayNG就可以订阅，测速也显示绿色了。

**Comment by millo0420** [comment]
现在 BPB Panel 采用 Release 形式发布 Worker.js 代码，这个方法还能同步最新的吗？

**Comment by runtu406076262** [comment]
不晓得耶，没试。你试了吗？



---Original---
From: ***@***.***&gt;
Date: Tue, Sep 9, 2025 19:28 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [leegical/leegical.github.io] posts/cf-bpb-vpn/ (Discussion #12)




 
现在 BPB Panel 采用 Release 形式发布 Worker.js 代码，这个方法还能同步最新的吗？

**Reply by millo0420** [reply]
好像每次发布新版本后，worker.js 路径都不一样，也无法拉取最新的代码，无法实现同步。不知道有什么办法没。

**Comment by runtu406076262** [comment]
我现在没用了，直接去找别人建立好的节点使用。也能撑挺久的。



---Original---
From: ***@***.***&gt;
Date: Tue, Sep 9, 2025 19:35 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [leegical/leegical.github.io] posts/cf-bpb-vpn/ (Discussion #12)




 
好像每次发布新版本后，worker.js 路径都不一样，也无法拉取最新的代码，无法实现同步。不知道有什么办法没。

**Comment by runtu406076262** [comment]
试了很多次，打算后面再看看



---Original---
From: ***@***.***&gt;
Date: Tue, Sep 9, 2025 19:35 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [leegical/leegical.github.io] posts/cf-bpb-vpn/ (Discussion #12)




 
好像每次发布新版本后，worker.js 路径都不一样，也无法拉取最新的代码，无法实现同步。不知道有什么办法没。

**Comment by okjx666** [comment]
检测真连接都是-1ms 测试ping是正常的 是哪里不对吗

**Reply by 2bsb384** [reply]
我也是这个问题，您解决了吗？

**Comment by 2bsb384** [comment]
再次部署的时候上传完了文件，它提示Your Pages project has been blocked.Contact
abusereply@cloudflare.com
这怎么解决呀？之前的步骤明明都成功了。谢谢了

**Reply by 2bsb384** [reply]
问题已解决，变量PROXYIP的值不能用bpb.yousef.isegaro.com，因为有关键词bpb（这玩意儿查的真严）

**Reply by sddyasdf** [reply]
PROXYIP的值是cdn-b100.xn--b6gac.eu.org，也出现our Pages project has been blocked.Contact abusereply@cloudflare.com的提示，无法创建部署。你确定只是变量值的事吗？

**Reply by Vpn2026Cloud9** [reply]
请问您是怎么解决的，需要买代理ip吗

**Reply by 2bsb384** [reply]
好像确实不只是变量值的事，但我把之前部署的page删了后重新弄了一个就成功了，并且很稳定，我还更新了一次bpb面板的版本呢。

**Comment by 2bsb384** [comment]
前面的步骤都没问题，但是订阅链接导入到软件,获取节点以后，用测试真连接测出来的连接速度是-1ms，实际也连不上。这个怎么解决呀？麻烦了，谢谢了。
